research/slim/nets/nets_factory.py

```diff
@@ -93,6 +93,34 @@
                   'nasnet_large': nasnet.nasnet_large_arg_scope,
                  }
 
+model_name_map = {'alexnet_v2': 'alexnet_v2',
+                  'cifarnet': 'CifarNet',
+                  'overfeat': 'overfeat',
+                  'vgg_a': 'vgg_a',
+                  'vgg_16': 'vgg_16',
+                  'vgg_19': 'vgg_19',
+                  'inception_v1': 'InceptionV1',
+                  'inception_v2': 'InceptionV2',
+                  'inception_v3': 'InceptionV3',
+                  'inception_v4': 'InceptionV4',
+                  'inception_resnet_v2': 'InceptionResnetV2',
+                  'lenet': 'LeNet',
+                  'resnet_v1_50': 'resnet_v1_50',
+                  'resnet_v1_101': 'resnet_v1_101',
+                  'resnet_v1_152': 'resnet_v1_152',
+                  'resnet_v1_200': 'resnet_v1_200',
+                  'resnet_v2_50': 'resnet_v2_50',
+                  'resnet_v2_101': 'resnet_v2_101',
+                  'resnet_v2_152': 'resnet_v2_152',
+                  'resnet_v2_200': 'resnet_v2_200',
+                  'mobilenet_v1': 'MobilenetV1',
+                  'mobilenet_v1_075': 'MobilenetV1',
+                  'mobilenet_v1_050': 'MobilenetV1',
+                  'mobilenet_v1_025': 'MobilenetV1',
+                  #'nasnet_cifar': 'nasnet_cifar',
```

Comment:
delete commented

Reply:
I have added it to complete code. If you sure of it, I will remove the comment.